README.md

```diff
@@ -17,6 +17,23 @@ More information about this project can be found at: https://jenkins.io/projects
 
 See the [CHANGELOG](CHANGELOG.md).
 
+## How to run demo of the plugin
+
+1. Requirements: docker, docker-compose installed.
+
+2. Build the images: `docker-compose build`.
+
+3. Run the demo script: `source run_demo.sh`.
```

Comment:
These steps could be replaced by  `make clean`, `make build` and `make run`. `make build` could also include building the plugin HPI. It is currently missing in these steps, and it seems that https://github.com/jenkinsci/remoting-kafka-plugin/blob/master/jenkins/Dockerfile actually cannot work in the recent version (broken relative path)